source/guides/routing/overview.md

```diff
@@ -76,7 +76,16 @@ get '/rack-app', to: 'rack_app' # it will map to RackApp.new
 
 ### Mounting Applications
 
-If we want to mount an application, we should use `mount`.
+If we want to mount an application, we should use `mount` within Hanami environment configuration file. The default configuration file is located at `config/environment.rb`. Place `mount` within the Hanami.configure block.
```

Comment:
I'm not sure what you are saying, it might be tied to something I'm unaware of but this is an attempt to solve your issue so I'm interested in your feedback.  Can you clarify?